janusgraph-test/src/main/java/org/janusgraph/diskstorage/IDAuthorityTest.java

```diff
@@ -101,16 +100,25 @@ public static ModifiableConfiguration getBasicConfig() {
     public KeyColumnValueStoreManager[] manager;
     public IDAuthority[] idAuthorities;
 
-    public final WriteConfiguration baseStoreConfiguration;
+    public WriteConfiguration baseStoreConfiguration;
 
-    public final int uidBitWidth;
-    public final boolean hasFixedUid;
-    public final boolean hasEmptyUid;
-    public final long blockSize;
-    public final long idUpperBoundBitWidth;
-    public final long idUpperBound;
+    public int uidBitWidth;
+    public boolean hasFixedUid;
+    public boolean hasEmptyUid;
+    public long blockSize;
+    public long idUpperBoundBitWidth;
+    public long idUpperBound;
 
-    public IDAuthorityTest(WriteConfiguration baseConfig) {
```

Comment:
As I see this block of code is executed only once before all test cases.
and current implementation execute this block of code individually for each case **after** `setUp`. It means that the current tests flow are different. I don't say that it is broken (didn't check it yet) but it is different. I will take another look on this either tomorrow or early next week. If you manage to fix it earlier it would be grate. Again, I didn't use JUnit 5 yet so I need some time to investigate this problem. 
I don't know if it is possible but can we use something like this?:
```
@BeforeAll
@MethodSource("configs")
public void open(WriteConfiguration baseConfig) throws BackendException {
    Preconditions.checkNotNull(baseConfig);
    TestGraphConfigs.applyOverrides(baseConfig);
    this.baseStoreConfiguration = baseConfig;
    blockSize = config.get(IDS_BLOCK_SIZE);
    idUpperBoundBitWidth = 30;
    idUpperBound = 1L <<idUpperBoundBitWidth;
}
```

Can you check it? If it works then we can remove `open(baseConfig);` from all tests and the flow fill be fixed.

Reply:
With Junit 5.5, We can do class wide parametrization.
https://github.com/junit-team/junit5/issues/878#issuecomment-354544841